backend/server.js

```diff
@@ -1,35 +1,36 @@
-const express = require('express');
-const cors = require('cors');
-const mongoose = require('mongoose');
+const express = require("express");
+const cors = require("cors");
+const mongoose = require("mongoose");
 
-require('dotenv').config();
+require("dotenv").config();
 
 const app = express();
-const port = process.env.PORT || 5000;
```

Comment:
you can leave the 5000 here. no need to force people to configure there port number